src/components/Common/Navigation/LogoWrap/LogoWrap.js

```diff
@@ -5,16 +5,19 @@
 import Logo from '../../../Logo'
 
 export default function LogoWrap({ children }) {
-  const brandName = <b className="font-weight-bold">Heathrow</b>
-
   return (
     <a
       className="navbar-brand d-none d-md-block"
       href="#/"
       onClick={(e) => e.preventDefault()}
     >
       {<Logo />}
-      <span>&nbsp; {brandName}</span>
+      <span>
+        &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
```

Comment:
um this madness requires explanation 😏

Reply:
It's just a way to add margin to the logo to avoid overlapping text like

![image](https://user-images.githubusercontent.com/58742260/160633177-78c37884-4bcd-4c01-ab62-dc2596562808.png)

Reply:
can we not add margin via styling?

Reply:
Done!